packages/frontend/src/pages/AttesterPage.tsx

```diff
@@ -0,0 +1,137 @@
+import { useContext, useState } from 'react'
+import { useParams } from 'react-router-dom'
+import { observer } from 'mobx-react-lite'
+import State from '../contexts/state'
+import Tooltip from '../components/info/Tooltip'
+import InfoCard from '../components/info/InfoCard'
+import AttestationCard from '../components/events/attesterEvents/AttestationCard'
+import UserCard from '../components/events/attesterEvents/UserCard'
+import Footer from '../components/layout/footer'
+
+
+export default observer(() => {
+  const { info } = useContext(State)
+  const { id } = useParams()
+  const [Selected, setSelected] = useState(0)
+  const toggleSelected = (Id: number) => setSelected(Id)
```

Comment:
Not a fan of functions like this. To me it's harder to understand code when there are wrapper functions used to rename things.

Reply:
thanks for pointing this out, yeah it's completely unnecessary here.